vehicle_safety:
  CTE_THR : 3 # cte threshold in meters
  CTE_THR_AT_CURVE : 5 # CTE Threshold at curve
  HEAD_THR : 10 # Heading Threshold in degrees(+/-)
  GPS_FIX_THR : 10.0 #GPS FIX Threshold time in seconds
  PX4_RESTART_SEC : 120 # Pixhawk restart delay to restart px4 if mavros behaves badly.[It only works if mavros GPS data is bad. It won't start mavros launch file]
  HEADING_TIME_THR : 10.0 # Heading Time threshold in seconds, if beyond this time, vehicle will stop
  BATT_TH : 40 # Battery Threshold in %. Below this value vehicle won't move
  STEER_DIFF_TH : 10 # Allowable Angle diff(degrees) from demand and current angle to detect steering stuck. Beware while changing this.
  STEER_STUCK_TIME_TH : 2 # Allowable time threshold(seconds) to detect steering stuck. Beware while changing this.
  WHITE_LIST : ["GPS","TrackingController","Emergency","GeoFence","CAN_Batt_Diagnostics"] # "mavros" # vehicle will stop,if these errors came up
  BLACK_LIST : ["Heading","CAN_steering_Diagnostics"] # vehcile will ignore and continue running, if the errors coming from these items.
  use_geo_fence : True
  geo_fence_coordinates: [
    [13.585564444929537, 79.96123159196975],
    [13.584881054339391, 79.9613978024903],
    [13.58421152386115, 79.96311317207108],
    [13.585489590870313, 79.96317221210154],
    [13.585564444929537, 79.96123159196975],
] #mango farm fence coordinates